src/ostree.scss

```diff
@@ -77,3 +83,21 @@
 .pf-v5-c-button.pf-m-link.pf-m-inline {
     padding-top: var(--pf-v5-c-button--PaddingTop);
 }
+
+.grey-color {
+  color: var(--pf-v5-global--Color--200);
+}
+
+.danger-color {
+  color: var(--pf-v5-global--danger-color--200);
+}
+
+.warning-color {
+  color: var(--pf-v5-global--warning-color--100);
+}
+
+.deployment-actions-btn {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+}
```

Comment:
There are also flex utility classes:

https://www.patternfly.org/utility-classes/flex

And the flex component itself can handle some of these things already, if not all: 

https://www.patternfly.org/layouts/flex

Reply:
right, this was some unused css I temporarily created